### ISSUE TITLE
Error Handling doesn't say how to get error/message

### ISSUE THREAD

**Issue by dubcanada** [mikecao/flight#30]
Hello,

http://flightphp.com/learn#error_handling

Doesn't say that you can use 

```
Flight::map('error', function(Exception $e){
    $e->getMessage();
    $e->getCode();
    $e->getTraceAsString();
});
```

to get error details.

**Comment by qlimax86** [comment]
Here you can find the definition of the Exception class:
http://php.net/manual/en/class.exception.php

**Comment by mikecao** [comment]
Fixed in commit 483ff648c472ca0aaf5ba426f780de3fcc68e080
